Added date and month to output

preactive opened the pull request:
As I am a swing, grave, day shift worker UTC will change days before local time does and its nice for a quick ref for "Monitor until 1-14 15:42 UTC" messages for next few crew on the following days.

https://i.imgur.com/cZJf6nc.png

preactive committed:
Added date and month to output

As I am a swing, grave, day shift worker UTC will change days before local time does and its nice for a quick ref for "Monitor until 1-14 15:42 UTC" messages for next few crew on the following days.

preactive commented:
Had an issue when around 15:00-1700 UTC the day tripped to the next day.

So changed to to something hacky in the mean time.  maybe you have some thoughts on why it does that?

Hacky UTC:
```
    var now = new Date();
    var hackyTime = now.getUTCMonth()+1 + "/" + now.getUTCDate() + " " + now.getUTCHours() + ":" + now.getUTCMinutes();
    label.set_text(hackyTime + ' UTC');
```

injcristianrojas commented:
Sorry for the (huge) delay. I have submitted a new version that has what you want to Extensions. Should be up in a few hours. Just go to the extension options and enable "show date"

preactive commented:
Awesome Sauce @injcristianrojas Thanks!  No worries about the delay.  Just thought about what has happened since I submitted this.  Changed companies, had a kid, Moved, and a host of other stuff.  Thank you for remembering about this.  :)
